.github/workflows/setupPythonVenv.yml

```
@@ -34,13 +40,19 @@ jobs:
     - name: Cache python virtualenv
       uses: syphar/restore-virtualenv@v1
       id: cache-venv
-      with:
-        requirement_files: '**/requirements*.txt'
       continue-on-error: true
 
     - uses: syphar/restore-pip-download-cache@v1
       if: steps.cache-venv.outcome == 'success' && steps.cache-venv.outputs.cache-hit != 'true'
 
-    - run: |
+    - name: Install dependencies via poetry
+      if: github.event.inputs.manager == 'poetry'
+      run: |
+        python -m pip install -U pip "setuptools<58" poetry
+        poetry install --no-root --no-interaction
+
+    - name: Install dependencies via pip
+      if: github.event.inputs.manager != 'poetry'
```

since we have only two options for inputs.manager. I suggest using the names of the options:
```suggestion
      if: github.event.inputs.manager == 'pip'
```

---

Unfortunately it is not going to work without adding input `manager=pip`  in all pipelines uses setipPythonVenv. I setup default='pip' but it does nothing. So value will be empty and == 'pip' is False. I would like to prevent updating all repos at least for now.